Flowchart/Controllers/ManageController.cs

```diff
@@ -419,20 +419,9 @@ private Task<ApplicationUser> GetCurrentUserAsync()
                     //    }
                     //}
 
-                    /// TODO: There will need to be some code here which will delete the flowchart
-                    /// data associated with the user which shouldn't be too hard at all.
-                    /// Simply gather all flowcharts which contain the user's username (email) and 
-                    /// delete them from the data after deleting the files on the hdd.
-
+                    await _signInManager.SignOutAsync();
```

Comment:
This will force the user to sign out prior to deleting them which is a simple work around for the user's logged-in status.